docker/Dockerfile

```diff
@@ -1,19 +1,19 @@
 FROM alpine:edge AS builder
-RUN apk add --no-cache crystal shards libc-dev \
-    yaml-dev libxml2-dev sqlite-dev zlib-dev openssl-dev \
-    sqlite-static zlib-static openssl-libs-static
 WORKDIR /invidious
 COPY ./shard.yml ./shard.yml
-RUN shards update && shards install
-RUN apk add --no-cache curl && \
+RUN apk add crystal shards libc-dev \
```

Comment:
This now requires all packages to be reinstalled once `shard.yml` changes, is this a good idea?

Reply:
oops, didn't thought about that. fixed